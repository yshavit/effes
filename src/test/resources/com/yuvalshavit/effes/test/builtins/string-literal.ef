main:
  debugPrint "hello there pooh bear"