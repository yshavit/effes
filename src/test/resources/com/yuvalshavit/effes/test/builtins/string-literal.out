"hello there pooh bear"
>> null